cirq-ft/requirements.txt

```diff
@@ -3,3 +3,5 @@ cachetools>=5.3
 ipywidgets
 nbconvert
 nbformat
+bitstring
+fixedpoint
```

Comment:
Can we double check that this is worth adding a new dependency? This looks somewhat stable but doesn't have many stars